dss-api.yml

```diff
@@ -1449,6 +1449,70 @@ paths:
                     enum: [unhandled_exception, Forbidden, Unauthorized, OAuthResponseProblem, not_found, read_only]
                 required:
                   - code
+  /bundles:
+    get:
+      operationId: dss.api.bundles.enumerate
+      summary: List through all bundles available to the data-store
+      description: >
+        Given a prefix
+      parameters:
+        - name: replica
+          in: query
+          description: Replica to fetch from.
+          required: true
+          type: string
+          enum: [aws, gcp]
+        - name: prefix
+          in: query
+          description: used to search for keys under a specific "bundle/" prefix
```

Comment:
Please provide a description that does not include the jargon words "keys" or "bundle/".

Reply:
```
- name: prefix
          in: query
          description: Is used to specify the beginning of a particular bundle UUID. Specifying a character(s) would provide all avaliable bundles starting with that character(s). 
          required: false
          type: string
```

how is this?